validate/validate.h

```diff
@@ -137,8 +139,17 @@ static inline bool IsHostname(const string& to_validate) {
 }
 
 static inline size_t Utf8Len(const string& narrow_string) {
-  std::wstring_convert<std::codecvt_utf8<char32_t>, char32_t> converter;
-  return converter.from_bytes(narrow_string).size();
+  const char *str_char = narrow_string.c_str();
+  ptrdiff_t byte_len = narrow_string.length();
+  size_t unicode_len = 0;
+  int char_len = 1;
+  while (byte_len > 0 && char_len > 0) {
+    char_len = google:protobuf:UTF8FirstLetterNumBytes(str_char, byte_len);
```

Comment:
@SecurityInsanity I'm missing something extraordinarily obvious here, a hint would be appreciated;
error C2882: 'google': illegal use of namespace identifier in expression

Reply:
Nevermind, needed coffee.